summaries/CICERO.md

```diff
@@ -0,0 +1,51 @@
+# Human-level play in the game of Diplomacy by combining language models with strategic reasoning
+
+Meta Fundamental AI Research Diplomacy Team (FAIR), Antin Bakhtun, Noam Brown, Emily Dinan **2022**
```

Comment:
Change order. See [template](https://github.com/vlgiitr/papers_we_read/blob/master/Summary_Template.md)

Reply:
Do you mean putting Meta team name after the authors' names? That is not the conference name if you misunderstood it for that.
They have stated it in the paper in this manner only, where they have first stated the team name and then the authors.

Reply:
Oh my bad ! Misunderstood that 😅